doc/nrf/gs_updating.rst

```diff
@@ -0,0 +1,202 @@
+
+You might also want to switch to a newer release or check out the latest state of development.
+However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
+For an overview of changes in the latest releases, see :ref:`release_notes`.
+
+.. _west_update:
+
+Updating west
+*************
+
+To update west, run the following command in the command window:
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install -U west
+
+   .. group-tab:: Linux
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install --user -U west
+
+   .. group-tab:: macOS
+
+      .. parsed-literal::
+         :class: highlight
+
+         pip3 install -U west
+
+..
+
+This command updates west to the latest available version in the PyPi repository.
+
+.. _gs_updating_repos:
+
+Updating the repositories
+*************************
+
+To manage the ``nrf`` repository (the manifest repository), use Git.
+To make sure that you have the latest changes, run ``git fetch origin`` to :ref:`fetch the latest code <dm-wf-update-ncs>` from the `sdk-nrf`_ repository.
+Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
+Running ``west update`` updates the project repositories to the state specified in this manifest file.
+
+.. include:: gs_installing.rst
+   :start-after: west-error-start
+   :end-before: west-error-end
+
+Examples of commands
+====================
+
+To switch to release |release| of the |NCS|, enter the following commands in the ``ncs/nrf`` directory:
+
+.. parsed-literal::
+   :class: highlight
+
+   git fetch origin
+   git checkout |release|
+   west update
+
+To update to a particular revision (SHA), make sure that you have that particular revision locally before you check it out (by running ``git fetch origin``)::
+
+   git fetch origin
+   git checkout 224bee9055d986fe2677149b8cbda0ff10650a6e
+   west update
+
+To switch to the latest state of development, enter the following commands::
+
+   git fetch origin
+   git checkout origin/master
+   west update
+
+.. note::
+   Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
+   This will bring the project repositories to the matching revision defined by the manifest file.
+
+.. _gs_updating_ses:
+
+Updating SEGGER Embedded Studio Nordic Edition
+**********************************************
+
+Each new release of the |NCS| might require manually updating to a newer version of the SEGGER Embedded Studio Nordic Edition.
+Whenever you update to a newer release of |NCS|, check the :ref:`gs_recommended_versions` page for the given release to see if you are using the minimum required version of the SES Nordic Edition.
```

Comment:
Product naming here as well. To use or not to use the definite article. No article in the heading, but in the text, we have it. Many other instances throughout the doc. Not only SES, but the NCS or NCS.

Reply:
I'll stick to what Jamie wrote on Teams and consider "nRF Connect" as the product/IP name, meaning I'll use "the" if there is any name that comes after.